test/specs/functional/keyboardNav.js

```diff
@@ -136,38 +147,20 @@ describe("Keyboard navigation functional browser driven tests", function() {
 			});
 		}
 
+		beforeEach(function() {
+			setMobileSize();
+		});
+
 		describe("Given I am on the NICE org site on a mobile and I click on the Menu button", function(){
 			it("should display the mobile menu", async function() {
 
-				setMobileSize();
-				browser.url("/example.niceorg.html"); // for base url see wdio.conf.js
+				browser.url("/example.niceorg.html");
 
 				browser.click("#menu-mobile");
 				browser.isVisible("#main-menu").should.be.true;
 			});
 		});
 
-		describe("Given I am on the NICE org site on a mobile", function(){
```

Comment:
Why have we removed this test as it specifically tests the mobile menu opposed to the full sized menu test above?

Reply:
It's just moved as I grouped them into sections by resolution... https://github.com/nhsevidence/NICE.TopHat/blob/131b548137c87ba8ad0278cbe35246a580e324c3/test/specs/functional/keyboardNav.js#L154